Gw2 Launchbuddy/MainWindow.xaml.cs

```diff
@@ -1914,12 +1914,12 @@ private void UpdateServerArgs()
 
         private void Window_LostKeyboardFocus(Object sender, System.Windows.Input.KeyboardFocusChangedEventArgs e)
         {
-            if (cinemamode || rb_cinemavideomode.IsChecked == true) { Cinema_MediaPlayer.Pause(); }
+            if (cinemamode | rb_cinemavideomode.IsChecked == true) { Cinema_MediaPlayer.Pause(); }
             BeginStoryboard(this.FindResource("anim_musicfadeout") as System.Windows.Media.Animation.Storyboard);
         }
         private void Window_GotKeyboardFocus(Object sender, System.Windows.Input.KeyboardFocusChangedEventArgs e)
         {
-            if (cinemamode || rb_cinemavideomode.IsChecked == true) { Cinema_MediaPlayer.Play(); } 
```

Comment:
No worries about documentation, sorry if I came off harsh!

So the | operator is very similar to ||. They both mean OR except that || stops running comparisons, while | will continue them. For a better explanation with examples you can look at [the microsoft docs about it](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-or-operator)

So the two operators in this case are effectively the same, since both operands are just value checks.